content/en/docs/faq.md

```diff
@@ -75,3 +75,9 @@ Not natively supported but a user can configure `revocationValidations` to `skip
 **A:** Ideally, we should validate the signing identity first and then use the public key in the signing identity to validate the artifact signature.
 However, this will lead to poor performance in the case where the signature is not valid as there are lots of validations against the signing identity including network calls for revocations, and possibly we won't even need to read the trust store/trust policy if the signature validation fails.
 Also, by validating artifact signature first we will still fail the validation if the signing identity is not trusted.
+
+## Notary Project Terms
+
+**Q: What is the difference between Notary, Notary v2, Notation, TUF, and Notary Project?**
```

Comment:
I would split this into a couple of questions. Here my proposal:
- What is Notary Project?
- How are Notary and Notary Project related?
- What is the difference between Notary Project and TUF?
- What is Notary Project specification?
- Does Notary Project signature specification leverage TUF?
- What is the difference between Notary and Notation?
- I've heard the term "Notary v2". What does this mean?

Reply:
This fine with me

Reply:
I have updated this PR to address Toddy's feedback except for this question: `Does Notary Project signature specification leverage TUF?` Could you please direct me to a document that could assist in clarifying the question?@toddysm  @zr-msft 
Thankk you.

Reply:
> I've heard the term "Notary v2". What does this mean?

@toddysm @zr-msft  For the term `Notary v2`, it's worth stating that this term was previously used with different meanings in many places but will not be used in the Notary Project since it is unclear enough. 

Because this term was widely used by others and is still visible in some articles (on the internet). To avoid ambiguous usage, we remind users that the term `Notary v2` is not valid anymore. Does it make sense? cc @Roseline-Bassey

Reply:
@Roseline-Bassey The answer for the following question is "No", and you can find what `Notary Project signature specification` is about from 4th paragraph of this PR https://github.com/notaryproject/.github/pull/32

```
Does Notary Project signature specification leverage TUF?
```

Reply:
Thanks @yizha1

Reply:
> > I've heard the term "Notary v2". What does this mean?
> 
> @toddysm @zr-msft For the term `Notary v2`, it's worth stating that this term was previously used with different meanings in many places but will not be used in the Notary Project since it is unclear enough.
> 
> Because this term was widely used by others and is still visible in some articles (on the internet). To avoid ambiguous usage, we remind users that the term `Notary v2` is not valid anymore. Does it make sense? cc @Roseline-Bassey

I think It'll be important to also mention some of the different interpretations that were/are associated with the term `Notary v2.` For example, For some, it represented the entirety of the project, and for others, it meant the notation CLI.  @FeynmanZhou

Reply:
> > I've heard the term "Notary v2". What does this mean?
> 
> @toddysm @zr-msft For the term `Notary v2`, it's worth stating that this term was previously used with different meanings in many places but will not be used in the Notary Project since it is unclear enough.
> 
> Because this term was widely used by others and is still visible in some articles (on the internet). To avoid ambiguous usage, we remind users that the term `Notary v2` is not valid anymore. Does it make sense? cc @Roseline-Bassey

I think It'll be important to also mention some of the different interpretations that were/are associated with the term `Notary v2.` For example, For some, it represented the entirety of the project, and for others, it meant the notation CLI.  @FeynmanZhou